gnd/src/main/java/com/google/android/gnd/repository/local/Edit.java

```diff
@@ -60,11 +60,19 @@ public static int toInt(@Nullable Edit.Type value) {
   @NonNull
   public abstract Type getType();
 
+  /**
+   * For edits of type UPDATE, returns a JSON object with the original value of modified attributes
```

Comment:
This might be as good a place as any to document how added and deleted attributes work. My understanding is:

* if a key is present in both `old_values` and `new_values`, the edit represents that the attribute has changed
* if a key is present in `old_values` but not `new_values`, the edit represents that the attribute is deleted
* if a key is present in `new_values` but not `old_values, the edit represents that the attribute is added

Reply:
Yes, that makes sense. Adding to Javadoc now.